FBC/FunctionalEntities/BusinessRegistries.rdf

```diff
@@ -342,15 +336,15 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>entity legal form registry entry</rdfs:label>
-		<skos:definition>an entry in an entity legal form registry that conforms to ISO 20275 and the Global Legal Entity Identifier Foundation (GLEIF) Entity Legal Form (ELF) code structure</skos:definition>
+		<skos:definition>entry in an entity legal form registry that conforms to ISO 20275 and the Global Legal Entity Identifier Foundation (GLEIF) Entity Legal Form (ELF) code structure</skos:definition>
```

Comment:
Don't need clause after "and" since GLEIF is the registration authority for ISO

Reply:
ok